layers/+lang/go/README.org

```diff
@@ -67,6 +67,7 @@ formatter, set the value of =gofmt-command=, e.g.
 | ~SPC m e b~ | go-play buffer                                            |
 | ~SPC m e r~ | go-play region                                            |
 | ~SPC m e d~ | download go-play snippet                                  |
+| ~SPC m e e~ | run "go run" for the current 'main' package               |
```

Comment:
what does `go run` do exactly ? It runs the program ?
If yes, then consider `mxx` instead of `mee` which is by convention `evaluate things under point` (in lisp it is evaluate last s-exp).